fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -273,6 +270,19 @@ public LocalMetastore(GlobalStateMgr globalStateMgr, CatalogRecycleBin recycleBi
         this.systemInfoService = systemInfoService;
     }
 
+    public static PartitionInfo buildPartitionInfo(CreateMaterializedViewStatement stmt) throws DdlException {
+        PartitionDesc partitionDesc = stmt.getPartitionExpDesc();
+        PartitionInfo partitionInfo;
+        if (partitionDesc != null) {
+            partitionInfo = partitionDesc.toPartitionInfo(
+                    Collections.singletonList(stmt.getPartitionColumn()),
+                    Maps.newHashMap(), false, false);
+        } else {
+            partitionInfo = new SinglePartitionInfo();
+        }
+        return partitionInfo;
+    }
+
```

Comment:
There is no real modification here, the code position is automatically moved by my IDE.